custom_components/tesla_custom/base.py

```diff
@@ -30,12 +30,6 @@ def __init__(
         self._attr_name = self.type.capitalize()
         self._attr_entity_registry_enabled_default = self._enabled_by_default
 
-    async def async_added_to_hass(self) -> None:
```

Comment:
This is already done in the BaseCoordinatorEntity 

https://github.com/home-assistant/core/blob/a25b2168a312713fafd73e769ab19f71a736ef6b/homeassistant/helpers/update_coordinator.py#L483